### ISSUE TITLE
Creating Custom Multiple Block Managers

### ISSUE THREAD

**Issue by rgernale** [artf/grapesjs#1492]
Is there a way to generate multiple block managers  where you click on a command button and a set on specific components are listed in the view panel? I recently asked about Multiple block mangers or filtering blocks based on criteria, but I should have explained a bit more.  Here is that I'm looking for. 
In the images attached - I have 4 panel views and 3 Category choices.  a Plane, a Graph and a Tag icon added to the panel.  I would like to choose each icon and show different views on screen.  When I've tried to recreate the Block Manager -  as a result, I would error out and have a blank screen to show in the browser.  If I can get help on this I would greatly appreciate it.

![button3](https://user-images.githubusercontent.com/5891727/46683558-83303700-cbbe-11e8-9215-1a0276a1263a.JPG)
![blockmanager](https://user-images.githubusercontent.com/5891727/46683559-83303700-cbbe-11e8-8ae0-03bbfb66eb70.JPG)
![button1](https://user-images.githubusercontent.com/5891727/46683560-83303700-cbbe-11e8-98b4-f451350508e4.JPG)
![button2](https://user-images.githubusercontent.com/5891727/46683561-83c8cd80-cbbe-11e8-8349-bbf4061f6e3e.JPG)
![block manager view](https://user-images.githubusercontent.com/5891727/46683994-a3142a80-cbbf-11e8-9edc-9b53470632c7.JPG)
![tag view](https://user-images.githubusercontent.com/5891727/46683995-a3142a80-cbbf-11e8-8875-5018676359f9.JPG)
![graph view](https://user-images.githubusercontent.com/5891727/46683996-a3142a80-cbbf-11e8-86e9-640caaa8ae35.JPG)
![plane view](https://user-images.githubusercontent.com/5891727/46683997-a3142a80-cbbf-11e8-80cc-be5154c1cc6d.JPG)

**Comment by alemenciones** [comment]
@rgernale hi bro, i'm trying to do the same as you, how did you manage to implement it?

**Comment by rgernale** [comment]
No. I haven’t had time to work on it. But I saw in one of the thread of something close. You’ll have just search
